starling.fsproj

```diff
@@ -76,19 +76,25 @@
     <Compile Include="Frontend.fs" />
```

Comment:
Not sure whether to do this as part of this prq or later?  Will look at tomorrow alongside need for additional tests.